### PR TITLE
Readme update - Astra connection quick start

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
Updates the Astra connectivity quick start code part of the README doc 

**Checklist**
- [X] Changes manually tested
- [X] Automated Tests added/updated
- [X] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,9 +129,7 @@ const mongoose = require("mongoose");
 const { driver, createAstraUri } = require("stargate-mongoose");
 
 const uri = createAstraUri(
-  process.env.ASTRA_DB_ID,
-  process.env.ASTRA_DB_REGION,
-  process.env.ASTRA_DB_KEYSPACE,
+  process.env.ASTRA_DB_API_ENDPOINT,
   process.env.ASTRA_DB_APPLICATION_TOKEN,
```
Comment (vkarpov15) [inline README.md:133]: Should also add `ASTRA_DB_NAMESPACE` as 3rd param

**Review by vkarpov15 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kathirsvn) [reply README.md:133]: I think it's an optional param and in Astra, the default namespace that gets created is `default_keyspace` which is connected to, by this code by default if we don't provide one explicitly.

Comment (vkarpov15) [reply README.md:133]: I know it's optional, but I think it is worthwhile to provide an example of how to set the URI's namespace. You can add `// optional` to indicate that the namespace argument is optional, but I think it is worth having.

Comment (kathirsvn) [reply README.md:133]: Sure, Fixed in this commit [4152374](https://github.com/stargate/stargate-mongoose/pull/179/commits/415237494f9c3d8e1d1175234ed6ecc0f5a0eebd).

**Review by vkarpov15 — APPROVED** [review]

**Pull request merged**